### PR TITLE
change updateStrategy to Strategy

### PR DESCRIPTION
**Pull request by prune998**
`updateStrategy` is only for Statefulsets, not Deployments

### CODE REVIEW

**Comment by mjpitz** [comment]
Resolves gh-229

**Comment by mjpitz** [comment]
Resolves gh-192

File: helm/nats-operator/values.yaml [hunk 1]
```diff
@@ -62,13 +62,12 @@ podAnnotations: {}
 ##
 podLabels: {}
 
-## Update strategy, can be set to RollingUpdate or OnDelete by default.
-## https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#updating-statefulsets
+## Update strategy, can be "Recreate" or "RollingUpdate". Default is RollingUpdate.
+
+updateStrategy: RollingUpdate
+# rollingUpdateMaxSurge: 25%
```
Comment (tokheim) [inline helm/nats-operator/values.yaml]: These probably shouldn't be commented out, or they will be set to empty in deployment

**Comment by kenschneider18** [comment]
Just ran into this issue myself, glad to see there's a PR already. Any clue when this will be prioritized?

**Comment by wallyqs** [comment]
@kenschneider18 sorry merging now...

**Pull request merged**